code/modules/power/cable.dm

```diff
@@ -27,7 +27,7 @@ By design, d1 is the smallest direction and d2 is the highest
 	anchored =1
 	var/datum/powernet/powernet
 	name = "power cable"
-	desc = "A flexible superconducting cable for heavy-duty power transfer"
+	desc = "A flexible superconducting cable for heavy-duty power transfer."

```

Comment:
superconducting works
there should be a comma between the two words; you're correct